Packs/FortinetFortiwebVM/ReleaseNotes/1_0_3.md

```diff
@@ -1,3 +1,4 @@
 #### Integrations
 ##### Fortiweb VM
-- Updated the Docker image to: *demisto/python3:3.10.10.49934*.
+- Updated the Docker image to: *demisto/python3:3.10.10.52956*.
```

Comment:
I don't see any docker update, can you check?
also - please add a description for the bug fix, e.g.:
```
- Fixed an issue where an unclear error message was returned in case of authentication failure.
```